app/indexers/concerns/hyku_indexing.rb

```diff
@@ -10,14 +10,15 @@ module HykuIndexing
     define_method method_name do |*args, **kwargs, &block|
       super(*args, **kwargs, &block).tap do |solr_doc|
         # rubocop:disable Style/ClassCheck
-        
+
         # specs refer to object as @object
         object ||= @object || resource
 
         solr_doc['account_cname_tesim'] = Site.instance&.account&.cname
         solr_doc['bulkrax_identifier_tesim'] = object.bulkrax_identifier if object.respond_to?(:bulkrax_identifier)
         solr_doc['account_institution_name_ssim'] = Site.instance.institution_label
         solr_doc['valkyrie_bsi'] = object.kind_of?(Valkyrie::Resource)
+        solr_doc['member_ids_ssim'] = object.member_ids.map(&:id)
```

Comment:
Note to self: We may need a valkyrie check here? valkyrie member ids returns valkyrie objects with id keys, allowing us to map (:id). Active Fedora's member_ids are just ids.